### ISSUE TITLE
Chevron PA Pattern fails to hit max flow , outer wall speeds changed AND not respected

### ISSUE THREAD

**Issue by psiberfunk** [SoftFever/OrcaSlicer#1616]
**Describe the bug**
This is a bit of a weird possible combination glitch, discovered when I realized the flow for the Ellis/Chevron PA pattern wasn't right by default.  There are two major issues here:

1. The defaults for the new PA pattern (Chevron style) for some reason changes the outer wall speed to 100 by default.  I'm not sure why this is done but it causes the pattern not to hit the max flow if VFR>~8 mm^3/sec on a standard 0.4 nozzle.  This is not a good default test setting as it is very common to go over this.  I'm not sure why the profile's outer wall speed should be altered for this test?

2.  The outer wall speed SHOULDNT actually break the test, but because somehow the slicer is applying the outer wall speed instead of the internal infill speed (which the slicer says the lines are on inspection), the test is broken and higher VFRs don't get tested right.

**To Reproduce**
Using this Win64 build:  https://github.com/SoftFever/OrcaSlicer/actions/runs/5693025113 :

1. Pick a filament profile with VFR=15, for example
2. setup the PA Pattern (chevron test) with defaults.
3. Slice. 
4. Observe that the max flow with a 0.4 Nozzle and bambu defaults shows a max flow of ~ 7.98 or so, as shown below in screenshots.
5. Switch to Line type mode.
6. Observe that somehow the outer wall speed is applying to lines marked as internal infill (purple).  This seems to be a bug too.

**Expected behavior**
The default test should not limit the max wall speed.  
Outer wall speed should not be applying to internal infill lines.

**Screenshots**
Max flow of ~7.98:

![image](https://github.com/SoftFever/OrcaSlicer/assets/19777571/a5148816-32c9-42bf-8dc0-9ce63106a75c)

Internal infill lines are getting outer wall speed limited:
![image](https://github.com/SoftFever/OrcaSlicer/assets/19777571/07fd8ae4-a471-4f10-b827-96f975fd12c6)

**Comment by psiberfunk** [comment]
Additional comment here:  I've also just discovered that the chevron test is not respecting the max volumetric flow limit settings in the filament setting.  To see this, set the max VFR for the current filament = 1, and re-slice... you will see that the flow view shows you that the flow far exceeds 1 !

@SoftFever  do you want a separate bug filed for this bad behavior or should it be handled in this ticket ?

**Comment by psiberfunk** [comment]
@SoftFever this is still real AFAIK

**Comment by psiberfunk** [comment]
This isn't stale, it's still a real issue.

**Comment by psiberfunk** [comment]
still not stale

**Comment by psiberfunk** [comment]
I don’t believe this is stale @SoftFever .. should it get marked as won’t do , or is it still considered ?

**Comment by MxBrnr** [comment]
I can confirm that in both 2.2.0-beta2 and in 2.2.0-RC, the PA Pattern test is still ignoring the filament's max vol. flow settings, and additionally ignoring line widths of less then 0.42mm (when using a 0.4mm nozzle), and ignoring travel acceleration. These two latter issues are not problems in the original Ellis3DP PA tuning tool.

**Comment by psiberfunk** [comment]
yes, this has long been an annoyance I wish would be remedied :)

**Comment by MxBrnr** [comment]
Also just discovered and confirmed with testing that the accels being used when actually printing the calibration are not following a standardized order. The first layer is printing using the first layer accel and speed (as it should be). The the second layer is printing using the outer wall speed (as it should) and the "normal printing" accel (odd, but ok), but then any subsequent layers are printing using the outer wall speed with the "Internal solid infill" accel (that's definitely not right). The accels noted here are also the ones being used for any travel moves during their respective layers. This is with Orca 2.2.0-RC

**Comment by psiberfunk** [comment]
I don’t think this is fixed still ?
